README.md

```diff
@@ -192,7 +192,7 @@ ESLint follows [semantic versioning](https://semver.org). However, due to the na
 
 According to our policy, any minor update may report more linting errors than the previous release (ex: from a bug fix). As such, we recommend using the tilde (`~`) in `package.json` e.g. `"eslint": "~3.1.0"` to guarantee the results of your builds.
 
-## Stylistic Rule Updates
+## Frozen Rule Updates
```

Comment:
I don't think this change is for the better. My assumption is explicitly making the section about _"stylistic"_ rules better surfaces what they're for. Someone skimming through the ToC and seeing a _"frozen rule updates"_ point won't be informed about why the rules are frozen.